src/transformers/utils/quantization_config.py

```diff
@@ -48,7 +49,7 @@ class QuantizationConfigMixin:
     quant_method: QuantizationMethod
 
     @classmethod
-    def from_dict(cls, config_dict, return_unused_kwargs=False, **kwargs):
+    def from_dict(cls, config_dict, return_unused_kwargs=False, config_origin="config", **kwargs):
```

Comment:
Can you elaborate more on the utility of `config_origin` argument? Do you think that we can find a way to overcome the usage of this argument?

Reply:
Ah yes that sounds great! Thanks for the suggestion, if you can make it with `quantization_status` usong an enum I think it will make it much more readble indeed!